src/core/network.js

```diff
@@ -414,6 +415,17 @@ PDFNetworkStreamFullRequestReader.prototype = {
       networkManager.abortRequest(fullRequestXhrId);
     }
 
+    // Content-Disposition: attachment; filename=Naïve file.txt
+    var fullRequestXhr = networkManager.getRequestXhr(fullRequestXhrId);
+    var contentDisposition = fullRequestXhr
```

Comment:
In the other parts of the codebase, we format this line as follows:

```javascript
var contentDisposition =
  fullRequestXhr.getResponseHeader('Content-Disposition');
```

Please do that here as well.